app/models/thredded/user_topic_follow.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Thredded
+  class UserTopicFollow < ActiveRecord::Base
+    REASON_MANUAL = 'manual'
```

Comment:
@glebm yes, yes I agree. Will see if I can find time to do (but trying to squeeze in other unrelated fixes and improvements)